dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -481,16 +478,20 @@ def _update_histogram(self, df_series):
             self._stored_histogram['histogram']['bin_edges'] = bin_edges
 
         # update loss for the stored bins
-        histogram_loss = self._histogram_bin_error(df_series)
+        try:
+            histogram_loss = self._histogram_bin_error(df_series)
+        except BaseException:
+            histogram_loss = None
```

Comment:
I'm not sure we should do this try here, as it may affect the later profile merging. Or, at least some warning message is needed. 
Thoughts @JGSweets @lettergram

Reply:
With the expected fix, is this still needed? Can other errors occur?

Reply:
Only other way would be values below the minimum index that would break `np.digitize`

Reply:
Not able to replicate with large negatives. I've removed the try/except block.